Cargo.toml

```diff
@@ -29,3 +29,6 @@ toml = "0.5.8"
 tracing = { version = "0", default-features = false }
 tracing-subscriber = "0"
 serde = { version = "1", default-features = false }
+
+[patch.'https://github.com/casper-network/casper-node']
+casper-types = { path = "../casper-node/types" }
```

Comment:
I think we can point to github now

Reply:
Updated in https://github.com/casper-network/casper-sidecar/pull/268/commits/ae07fbc8c155bc7a9056a41a0b3088c6514598e4